doc/plugins/server-info.md

```diff
@@ -0,0 +1,104 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+- [中文](../zh-cn/plugins/server-info.md)
+
+# Summary
+
+- [**Name**](#name)
+- [**Attributes**](#attributes)
+- [API](#api)
+- [**How To Enable**](#how-to-enable)
+- [**Test Plugin**](#test-plugin)
+- [**Disable Plugin**](#disable-plugin)
+
+## Name
+
+`server-info` is a plugin that reports basic server information to etcd periodically.
+
+The meaning of each item in server information is following:
+
+| Name    | Type | Description |
+|---------|------|-------------|
+| up_time | integer | Elapsed time since APISIX instance was launched, value will be reset when you hot updating APISIX but is kept for intact if you just reloading APISIX. |
+| boot_time | integer | Bootstrap timestamp of the APISIX instance, value will be reset when you hot updating APISIX but is kept for intact if you just reloading APISIX. |
```

Comment:
I think the value is different, as different workers write data to the same shdict.

Reply:
Sorry i didn't capture your meaning.

Reply:
When reloading Nginx, there will be two workers A and B. B is just created and has a later boot time, but both workers share the same shdict.

Reply:
Previously, workers will try to write server info to shdict in the init_worker phase, only when the key "server_info" doesn't exist, so server info written by old worker will not be overwritten by new workers, that is to say, the boot time will not be changed.

Reply:
> only when the key "server_info" doesn't exist

So where is the check? Do I miss something?

Reply:
Now the logic about writing the uninitialized data was removed.

since after APISIX launches, timer will be created immediately which updates the server info according to existing one in shdict, the boot_time will be inherited (or the uninitialized data if no data in shdict).